io.github.syco.blink.json

```diff
@@ -0,0 +1,406 @@
+{
+  "id": "io.github.syco.blink",
+  "command": "/app/bin/blink",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "5.14",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--talk-name=org.kde.StatusNotifierWatcher"
+  ],
+  "rename-desktop-file": "blink.desktop",
+  "modules": [
+    {
+      "name": "cpython",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/python/cpython/archive/v2.7.17.tar.gz",
+          "sha256": "d2b73d2d2607ec3f67bc10cef2d383008f95b835150d97cd00e55283d4afe360"
+        }
+      ]
+    },
+    {
+      "name": "pip",
+      "modules": [
+        {
+          "name": "mpfr_for_gmpy2",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.gz",
+              "sha256": "ae26cace63a498f07047a784cd3b0e4d010b44d2b193bab82af693de57a19a78"
+            }
+          ]
+        },
+        {
+          "name": "mpc_for_gmpy2",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://ftp.gnu.org/gnu/mpc/mpc-1.1.0.tar.gz",
+              "sha256": "6985c538143c1208dcb1ac42cedad6ff52e267b47e5f970183a3e75125b43c2e"
+            }
+          ]
+        }
+      ],
+      "buildsystem": "simple",
+      "build-options": {
+        "build-args": [
+          "--share=network"
```

Comment:
That's not possible. Builds are without network access on Flathub, you should bundle everything for reproducibility

Reply:
But python packages have lots of dependencies, is there a better way than listing them one by one in the json file?